### PR TITLE
Feedback fixes

### LINKED ISSUE #42: [FEATURE] - Mixed MVP Fixes
### Reference Issues

_No response_

### Summary

Mixed feedback :)

### Unresolved questions

_No response_

### Definition of Done

- [x] - border om titel og description
- [x] - description fyld ud
- [x] - loading knap ting når man submitter
- [x] - validering test template parameter
- [x] - popup grøn confirm
- [x] - popup kun vis hvis der er test cases
- [x] - fjern input dropdown fra edit test case
- [x] rest test cases on successful create exercuise
- [x] - bredde af dropdowns forskel
- [x] - gør forskellen på typer og værdier mere tydelig e.g. fed skrift for types
- ~~[ ] - skriv input/output som et "kolonnenavn" så man ikke skriver det samme flere steder~~
- [x] Float -> Floating Point
- [x] Opdater haskell boilerplate så floating points er Double og ikke Float
- [x] Brug goto istedet for href
- [x] slet kommenteret kode

### PR DESCRIPTION
**Pull request by andreaswurtz**
## Description

Feedback after showing mvp to the boys

### Resolved Issue
Fixes #42 

## Changes

See issue.

### CODE REVIEW

File: src/routes/session/[id]/+layout.svelte [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<script>
+	export let data;
+</script>
+
+<slot />
+
+<style global>
+	* {
+		transition: none !important;
```
Comment (EMojoKrogh) [inline src/routes/session/[id]/+layout.svelte:9]: please no

File: static/favicon.jpg
Comment (EMojoKrogh) [inline static/favicon.jpg:1]: 🦞

**Review by EMojoKrogh — CHANGES_REQUESTED** [review]
Look comment

File: src/routes/session/[id]/+layout.svelte (hunk 1 above)
Comment (andreaswurtz) [reply src/routes/session/[id]/+layout.svelte]: ok arrow up

**Review by EMojoKrogh — APPROVED** [review]
Hihi

**Pull request merged**